app/services/organization_service.rb

```diff
@@ -0,0 +1,12 @@
+class OrganizationService
+  def save_organization_and_cities_organization(organization: Organization.new,
+                                                 city_ids: [])
+    return false unless organization.valid?
+
+    organization.save
+
+    city_ids.each do |city_id|
+      CitiesOrganization.create(city_id: city_id, organization_id: organization.id)
+    end
+  end
+end
```

Comment:
You don't even need this service. Rails will do all of these automatically with properly setuped associations. Also check out how are Forms created in Rails https://guides.rubyonrails.org/form_helpers.html